### PR TITLE
Feature/iprobe and logging update

### PR DESCRIPTION
**Pull request by shuds13**
Loop iprobe to ensure receive messages. Also add timestamp to logger and make some debug logging updates.

### CODE REVIEW

**Comment by shuds13** [comment]
Is the gen_support logging overkill?

File: libensemble/manager.py [hunk 1]
```diff
@@ -426,14 +428,17 @@ def _handle_msg_from_worker(self, persis_info, w):
...
             if not self.WorkerExc:
                 self.WorkerExc = True
                 self._kill_workers()
                 raise WorkerException('Received error message from worker {}'.format(w),
                                       D_recv.msg, D_recv.exc)
         elif isinstance(D_recv, logging.LogRecord):
+            logger.debug("Manager received a log message from worker {}".format(w))
             logging.getLogger(D_recv.name).handle(D_recv)
         else:
+            logger.debug("Manager received data message from from worker {}".format(w))
```
Comment (jlnav) [inline libensemble/manager.py:441]: extra "from"

Comment (shuds13) [reply libensemble/manager.py:441]: Thanks!

**Review by jlnav — APPROVED** [review]

**Review by jmlarson1 — APPROVED** [review]

**Comment by jmlarson1** [comment]
Maybe add a simple text comment as to why the loop around iprobe was added.

**Pull request merged**